README.md

```diff
@@ -457,7 +457,12 @@ Instead of using `dotnet publish` to deploy changes to a database, you can also
 </Project>
 ```
 
-With this enabled you'll find a SQL script with the name `<datbase-name>_Create.sql` in the bin folder.
+With this enabled you'll find a SQL script with the name `<database-name>_Create.sql` in the bin folder.
+The database name for the create script gets resolved in the following manner:
+1. `TargetDatabaseName`.
```

Comment:
Isn't this either `TargetDatabaseName` or the name of the project file? I believe those are the only two options right now looking at this change right?

Reply:
There is a third condition in `PackageBuilder.GenerateCreateScript` for when the passed in name is null. I don't think that can ever happen, as the package name is always set via `MetadataArguments`. I propose:
- remove the third option from the readme, and
- remove the condition from GenerateCreateScript.

Reply:
I guess from the point of view of the command line tool it could possible be null, even though `Sdk.targets` will always provide it. If someone would want to run the command line tool directly it would be nice if they'd now that `--name` is a required argument. I'm fairly sure that System.CommandLine has something for that. If we add that, I'm fine with removing the fallback.

Reply:
@AroglDarthu Just checking if you saw my comment here ;).

Reply:
I looked into `System.CommandLine`. The docs don't mention `IsRequired`, but fortunately someone filed an [issue](https://github.com/dotnet/command-line-api/issues/1075) about the docs not mentioning that feature 🙈